app/services/submissions/update_service.rb

```diff
@@ -21,6 +21,7 @@ def add_events(submission, params, save: false)
           event["updated_at"] ||= event["created_at"]
 
           submission.events << event.as_json
+          submission.last_updated_at = event["created_at"]
```

Comment:
I think we probably want safeguards that when an array of events is passed in, we use the maximum `created_at`. There doesn't seem to be anything explicit in the payload construction on the caseworker end that the most recent event will be at the end of the array.